content/en/docs/faas/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Cloud Functions
+description: >-
+  OpenTelemetry supports various methods of monitoring Cloud Functions
+weight: 2
+---
+
+Functions as a Service (FaaS) is an important serverless compute platform for cloud native applications. However, platform quirks usually mean these applications have slightly different monitoring guidance and requirements than applications running on Kubernetes or Virtual Machines.
+
+The initial vendor scope of the FaaS documentation is around Azure, GCP, and AWS functions (lambdas).
+
+### Community Assets
+
+The community currently provides pre-built Lambda layers to quickly add instrumentation or the Collector to your Lambda.
```

Comment:
This statement is a bit strange. 

> or the Collector to your Lambda.

I think you can say something like:

> The community currently provides pre-built Lambda layers able to auto-instrument your application as well as a the option of standalone collector lambda layer that can be used when instrumenting applications manually.
your Lambda.

Reply:
Tweaked the final wording slightly but updated